src/panels/lovelace/cards/hui-picture-glance-card.ts

```diff
@@ -318,6 +318,11 @@ class HuiPictureGlanceCard extends LitElement implements LovelaceCard {
         padding-bottom: 4px;
         padding-top: 4px;
       }
+      ha-icon:focus {
+        outline: none;
+        box-shadow: 0 0 2px 2px #51a7e8;
+        border-radius: 100px;
```

Comment:
We should stay closer to material, so I suggest something like this: (not sure if it is the correct color and if we have a good variable for it)
```css
        outline: none;
        background: var(--divider-color);
```

Reply:
![image](https://user-images.githubusercontent.com/1287159/68030666-ff912e80-fc87-11e9-9d62-3bc7ea2cfa39.png)
yeah that looks close to what you get on mwc-button

Reply:
haven't come up with anything good looking for cards or state-label-badge (found in badges, glance and picture-elements) yet